web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -123,6 +132,67 @@ be edited, its `Revision: X` label will be increased by one, and a note will be
 to the `PDEP-N history` section. This will let readers understand that the PDEP has
 changed and avoid confusion.
 
+## <a id="List-of-issues"></a> List of issues that could have been PDEPs for context
+### Clear examples for potential PDEPs:
+- Adding a new parameter to many existing methods, or deprecating one in many places. For example:
```

Comment:
Some markdown parsers require a blank line before lists. Ours doesn't, but for other times maybe good to leave it in case we ever change to a more strict one.